cheroot/test/test_server.py

```diff
@@ -236,3 +237,65 @@ def test_peercreds_unix_sock_with_lookup(peercreds_enabled_server):
         peercreds_text_resp = requests.get(unix_base_uri + PEERCRED_TEXTS_URI)
         peercreds_text_resp.raise_for_status()
         assert peercreds_text_resp.text == expected_textcreds
+
+
+@pytest.mark.skipif(
+    IS_WINDOWS,
+    reason=(
+        'This regression test is for a Linux bug, and the resource module '
+        'is not available on Windows.'
+    ),
+)
+def test_high_number_of_file_descriptors():
```

Comment:
I think, we need to isolate this test with `pytest-forked`
```suggestion
@pytest.mark.forked
def test_high_number_of_file_descriptors():
```